nRF51/rbc_mesh/src/timer.c

```diff
@@ -99,8 +103,11 @@ void timer_event_handler(void)
         if (NRF_TIMER0->EVENTS_COMPARE[i])
         {
             NRF_TIMER0->EVENTS_COMPARE[i] = 0;
+#ifdef RACE_CONDITION_SOLVED
```

Comment:
We've several other issues as well, but I first wanted to address the obvious changes. I didn't change the name of the nRF51 folder, so it's easier to merge.